src/platform_linux/platform_inline.h

```diff
@@ -150,22 +150,32 @@ platform_semaphore_try_wait(platform_semaphore *sema)
 }
 
 static inline platform_status
-platform_mutex_lock(platform_mutex *mu)
-{
-   int ret;
-
-   ret = pthread_mutex_lock(mu);
+platform_mutex_lock(platform_mutex *lock)
+{
+   int ret = pthread_mutex_lock(&lock->mutex);
+   platform_assert(
+      lock->owner == INVALID_TID,
+      "Locked a mutex with an existing owner, lock: %p, tid: %lu, owner: %lu\n",
```

Comment:
I think this message wants to say: "Found a locked mutex ..."

And I found that this formatting keeps clang-format happy, while improving readability:

```
156    platform_assert(lock->owner == INVALID_TID,
157                    "Found a locked a mutex with an existing owner"
158                    ", lock: %p, tid: %lu, owner: %lu\n",
```